app/views/dashboard/_calendar.html.erb

```diff
@@ -4,7 +4,32 @@
 
   <% entries.each do |entry| %>
     <div>
-      <%= entry.mood %>
+      <% if entry.mood %>
+        <button class='emoji-btn' type="button" name="button" data-toggle="modal" data-target="#modal<%= date.strftime("%d") %>"><%= entry.mood %></button>
+      <% end %>
+    </div>
+
```

Comment:
Does the date in this button need to be `entry.date.strftime("%d")` or does just `date.strftime("%d")` get extracted automatically from the calendar?

Reply:
Daniel pointed out that it is coming from a different block not shown here in the files changed view!